cinema/views.py

```diff
@@ -183,7 +212,7 @@
     )
     serializer_class = OrderSerializer
     pagination_class = OrderPagination
-    authentication_classes = (TokenAuthentication,)
+    # authentication_classes = (TokenAuthentication,)
```

Comment:
The `authentication_classes` are commented out. Ensure this is intentional, as it might leave the endpoints unprotected. Consider enabling them for production to secure the API.